src/server/config/passport-setup.js

```diff
@@ -0,0 +1,43 @@
+import passport from 'passport';
+import GoogleStrategy from 'passport-google-oauth20';
+import keys from './keys';
+import DB from '../models/index';
+
+const { User } = DB;
+
+passport.serializeUser((user, done) => {
+  done(null, user.id);
+});
+
+passport.deserializeUser((id, done) => {
+  User.findByPk(id).then(user => {
+    done(null, user);
+  });
+});
+
+passport.use(
+  new GoogleStrategy(
+    {
+      clientID: keys.google.clientID,
+      clientSecret: keys.google.clientSecret,
+      callbackURL: '/auth/google/redirect',
+    },
+    (accessToken, refreshToken, profile, done) => {
+      User.findOne({ where: { googleId: profile.id } })
+        .then(user => {
+          if (user) {
+            return user;
+          }
+          return User.create({
+            googleId: profile.id,
+            email: profile.emails[0].value,
+            name: profile.displayName,
+            img: profile.photos[0].value,
+          });
+        })
+        .then(user => {
+          done(null, user);
+        });
```

Comment:
Move this statements to User's model functions, like `findUserByGoogleId()` and `createUser()`.